Gems/ROS2/Code/Source/ContactSensor/ROS2ContactSensorComponent.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include "AzCore/Component/EntityId.h"
+#include "AzCore/Math/Vector3.h"
+#include "AzCore/RTTI/ReflectContext.h"
+#include "AzCore/std/containers/unordered_map.h"
+#include "AzCore/std/parallel/mutex.h"
+#include "AzCore/std/string/string.h"
+#include "AzFramework/Physics/Common/PhysicsSimulatedBodyEvents.h"
+#include "ROS2/Sensor/ROS2SensorComponent.h"
+#include <gazebo_msgs/msg/contact_state.hpp>
+#include <gazebo_msgs/msg/contacts_state.hpp>
+#include <memory>
+#include <rclcpp/publisher.hpp>
+#include <unordered_map>
```

Comment:
You already use and include AZStd version instead. Check includes please.

Reply:
Fixed